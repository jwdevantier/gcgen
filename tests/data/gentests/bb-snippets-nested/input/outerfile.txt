
My outer file...

First foo:
# [[start foo
# end]]

Then bar:
# [[start bar
# end]]