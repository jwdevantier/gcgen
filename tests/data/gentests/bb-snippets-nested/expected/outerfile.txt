
My outer file...

First foo:
# <<? foo ?>>
foo from outer ctx!
scope["some-var"]: <<some-outer-val>>
/outer foo end
# <<? /foo ?>>

Then bar:
# <<? bar ?>>
bar from outer ctx!
# <<? /bar ?>>